mappings/net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig.mapping

```diff
@@ -1,5 +1,8 @@
 CLASS net/minecraft/class_2906 net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig
+	FIELD field_13221 seaLevel I
 	FIELD field_13222 forcedBiome I
+	FIELD field_13223 riverSize I
+	FIELD field_13224 biomeSize I
```

Comment:
```suggestion
	FIELD field_13222 forcedBiome I
```
These are incorrect, see https://github.com/FabricMC/yarn/pull/1123#pullrequestreview-366511961

Reply:
I disagree and would like even more background 
I think that forcedBiome is wrong. and needs to propagate.

Reply:
The thing is that these fields are unmapped, and there's no way to know which field belongs to which getters -- and one (`field_13221`) doesn't even match any method in that class.

Reply:
I agree with what you are saying about the pointlessness of the private member's. What I don't agree with is leaving it as is as that will just make issues like this happen over and over again. 
Here is a committed suggestion on how to deal with this. 
https://github.com/FabricMC/yarn/pull/1218/commits/408beee58681a59fd988416744f4cbf171a7e16d

Reply:
They should not be named if there's no way to confirm what those fields are, like in this case.

Reply:
Well there is one way to ensure that they are.....

Reply:
There's not enough evidence for those. See #1215 and #1123. It's best to leave those unmapped for now, and remove the javadoc.

Reply:
What I am really saying is that you all are going to have to deal with this over and over again. dev's are going to see something that is un-named and go after it. That is what dev's do. I think that it does not matter at all if they are named or not. as we know what is going on here with the injection from javac.  Things we do know for sure is that 
seaLevel is in fact y=63

maybe they should be used as something like 
```
    private final int unused_riverSize = 4;
```